apps/sms/views/shared/js/navigation.js

```diff
@@ -106,7 +105,7 @@ var Navigation = {
     var currentArgs = currentPanel.args || {};
 
     for (var arg in args) {
-      if (args[arg] !== currentArgs[arg]) {
+      if (args[arg] != currentArgs[arg]) {
```

Comment:
That's the problem I don't have good solution yet - in views we use  ```isCurrentPanel``` with _number_ thread id, but when we load panel separately and pass thread id as parameter it's parsed as string...

I was thinking about JSONable query string parameters that will solve this, but it's kind of ugly... Don't know really.

What do you think?